### ISSUE TITLE
Numbers on ionic run ios

### ISSUE THREAD

**Issue by olostan** [driftyco/ionic-cli#1078]
When running application, strange numbers starts to appear instead of output:
```
➢ ionic run ios --device
(node:11780) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.

Running 'run:before' gulp task before run
[04:54:33] Starting 'clean'...
[04:54:33] Finished 'clean' after 44 ms
[04:54:33] Starting 'build'...
[04:54:33] Starting 'sass'...
[04:54:33] Starting 'html'...
[04:54:33] Starting 'fonts'...
[04:54:33] Starting 'scripts'...
[04:54:33] Finished 'html' after 55 ms
[04:54:33] Finished 'scripts' after 53 ms
[04:54:33] Finished 'fonts' after 69 ms
[04:54:34] Finished 'sass' after 992 ms
[04:54:41] Finished 'build' after 7.89 s
[04:54:41] Starting 'run:before'...
[04:54:41] Finished 'run:before' after 6.93 μs

 0=82, 1=117, 2=110, 3=110, 4=105, 5=110, 6=103, 7=32, 8=99, 9=111, 10=109, 11=109, 12=97, 13=110, 14=100, 15=58, 16=32, 17=47, 18=85, 19=115, 20=101, 21=114, 22=115, 23=47, 24=118, 25=115, 26=104, 27=121, 28=98, 29=97, 30=110, 31=111, 32=118, 33=47, 34=99, 35=111, 36=100, 37=101, 
38=47, 39=105, 40=111, 41=110, 42=105, 43=99, 44=47, 45=119, 46=97, 47=108, 48=108, 49=101, 50=116, 51=99, 52=111, 53=97, 54=99, 55=104, 56=47, 57=104, 58=111, 59=111, 60=107, 61=115, 62=47, 63=97, 64=102, 65=116, 66=101, 67=114, 68=95, 69=112, 70=114, 71=101, 72=112, 73=97, 74=114, 75=101, 76=47, 77=48, 78=49, 79=48, 80=95, 81=97, 82=100, 83=100, 84=95, 85=112, 86=108, 87=97, 88=116, 89=102, 90=111, 91=114, 92=109, 93=95, 94=99, 95=108, 96=97, 97=115, 98=115, 99=46, 100=106, 101=115, 102=32, 103=47, 104=85, 105=115, 106=101, 107=114, 108=115, 109=47, 110=118, 111=115, 112=104, 113=121, 114=98, 115=97, 116=110, 117=111, 118=118, 119=47, 
120=99, 121=111, 122=100, 123=101, 124=47, 125=105, 126=111, 127=110, 128=105, 129=99, 130=47, 131=119, 132=97, 133=108, 134=108, 135=101, 136=116, 137=99, 138=111, 139=97, 140=99, 141=104
```

```
➢ ionic info

Your system information:

Cordova CLI: 6.2.0
Ionic Framework Version: 2.0.0-beta.7
Ionic CLI Version: 2.0.0-beta.30
Ionic App Lib Version: 2.0.0-beta.16
ios-deploy version: 1.8.6 
ios-sim version: 5.0.8 
OS: Mac OS X El Capitan
Node Version: v6.2.1
Xcode version: Xcode 7.3.1 Build version 7D1014 
```

**Comment by MehdiZonjy** [comment]
I'm having the same problem on Ubuntu  16.04 targeting  Android platform. I don't think it's a platform specific issue.

`> ionic platform list 0=73, 1=110, 2=115, 3=116, 4=97, 5=108, 6=108, 7=101, 8=100, 9=32, 10=112, 11=108, 12=97, 13=116, 14=102, 15=111, 16=114, 17=109, 18=115, 19=58, 20=10, 21=32, 22=32, 23=97, 24=110, 25=100, 26=114, 27=111, 28=105, 29=100, 30=32, 31=53, 32=46, 33=49, 34=46`

I'm getting a similar output using different commands such as
```

> ionic run android --device
> ionic platform list
> ionic plugin list

```

Here is my system info

```
>ionic info
Your system information:
Cordova CLI: 6.2.0
Gulp version:  CLI version 3.9.1
Gulp local:   Local version 3.9.1
Ionic Framework Version: 2.0.0-beta.8
Ionic CLI Version: 2.0.0-beta.30
Ionic App Lib Version: 2.0.0-beta.16
OS: Distributor ID:	Ubuntu Description:	Ubuntu 16.04 LTS 
Node Version: v6.2.1

```

**Comment by sfabriece** [comment]
Same on windows 10

```
Your system information:

You have been opted out of telemetry. To change this, run: cordova telemetry on.
6.2.0

Gulp version:  CLI version 3.9.1
Gulp local:   Local version 3.9.1
Ionic Framework Version: 2.0.0-beta.9
Ionic CLI Version: 2.0.0-beta.31
Ionic App Lib Version: 2.0.0-beta.17
OS:
Node Version: v6.2.2

```.

**Comment by galuszkak** [comment]
I have the same problem described above on OS X.

**Comment by dmastag** [comment]
I created #1080 which looks like a duplicate of this ticket. After some checking I found that it is working for me using node < 6.2.1 as explained in #1080

**Comment by sfabriece** [comment]
any plans to fix this?

**Comment by dmastag** [comment]
@sfabriece node 6 is not officially supported right now and I think that there are other more important things for the ionic team right now like getting the ionic2 out of beta.

**Comment by sfabriece** [comment]
no arguments here, I can wait.
Thanks @dmastag

**Comment by jthoms1** [comment]
Found the issue. Just committed the fix 5b400e712947ec1dff4e92d8fa53668cdd297568.  This should be released soon.

**Comment by galuszkak** [comment]
Awesome! Thanks @jthoms1 !
